packages/@aws-cdk/integ-tests/test/integ.test-case.ts

```diff
@@ -0,0 +1,11 @@
+import * as cdk from '@aws-cdk/core';
+import { IntegTest } from '../lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'integ-tests-integ');
+
+new IntegTest(app, 'IntegTest', {
```

Comment:
Instead of adding a test here, can you update one of the existing integration tests to use this new method?

For example, this one would be a good one to update.
https://github.com/aws/aws-cdk/blob/cde8bb91a303a17e2820596265ab085707c2a06e/packages/@aws-cdk/aws-lambda-destinations/test/integ.destinations.ts#L87-L98

Reply:
Agreed, I'll work on that.
I selected another test file to update https://github.com/aws/aws-cdk/blob/df163ec72c61b521ce6f8d7555872c1342a70745/packages/%40aws-cdk/aws-lambda/test/integ.lambda.docker.ts#L16

Indeed, I did not manage to get a green test result on the test you pointed on `main` branch, for unknown reason :s